### PR TITLE
Changed app icon to be in svg format

### PR DESCRIPTION
**Pull request by Mihaiii**
For https://github.com/Starcounter/Guidelines/issues/41

### CODE REVIEW

File: src/Products/wwwroot/Products/images/Products.svg [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 20010904//EN"
+ "http://www.w3.org/TR/2001/REC-SVG-20010904/DTD/svg10.dtd">
+<svg version="1.0" xmlns="http://www.w3.org/2000/svg"
+ width="64.000000pt" height="64.000000pt" viewBox="0 0 64.000000 64.000000"
+ preserveAspectRatio="xMidYMid meet">
+  <symbol id="css-file">
```
Comment (Mackiovello) [inline src/Products/wwwroot/Products/images/Products.svg]: The `Id` here might be dangerous

**Comment by Mihaiii** [comment]
Yes, it should.
There are some blending related problems and the icons aren't displayed in Launchpad with latest RC (they are displayed with some previous version - will look it up if you want).

**Comment by Mihaiii** [comment]
@Mackiovello What version of SC are you using? 
I tried it with 2.3.1.7334 (current latest RC) and the problem seems to be gone.

**Comment by Mackiovello** [comment]
I'm on the same version, but I used the website and launchpad from the warehouse.

I'll try with the develop branches

**Comment by Mackiovello** [comment]
With Launchpad from the develop branch it works

**Pull request merged**